### DISCUSSION TITLE
Hide type hint

### DISCUSSION THREAD

**Discussion by ziulev** [hrsh7th/nvim-cmp#581] [Q&A]
I haven't found this in the documentation.
Is there a way to disable hint e.g. `(method) Console.log(...data: any[]): void` ?

![image](https://user-images.githubusercontent.com/14956407/143588165-6b01cbf6-aeb9-4799-8748-419f1c74a817.png)

**Comment by Iamafnan** [comment]
You can disable documentation like this

```lua
require('cmp').setup({
    documentation = false,
})
```
